doc/html_publisher.rst

```diff
@@ -0,0 +1,44 @@
+Integrate collapsible log into Jenkins
+======================================
+
+The `--html-log <args.html#Output>`__ option enables generating an HTML page with collapsible blocks,
+very convenient when navigating a long build log of a `Universum` run with many steps.
+
+While TeamCity has its own mechanisms increasing readability, and Universum ran locally by default just stores
```

Comment:
This is a really long sentence, which is hard to parse.
Split into multiple shorter ideas. E.g.
By default, Jenkins outputs full build log as a plain text without navigation anchors. To simplify interaction and debug, we provide a way to integrate generation of a user-friendly log into a Universum job.
Note: this mechanism is not needed for certain use cases. TeamCity has its own mechanism blabla. Running Universum locally blabla.